spec/features/admin_admin_set_spec.rb

```diff
@@ -15,10 +16,7 @@
     login_as(user, scope: :user)
     visit '/dashboard'
     click_link "Administrative Sets"
-
-    expect(page).to have_link "Create new administrative set"
```

Comment:
The next line says to click that link. If it wasn't present the "click_link" would fail. So this is a duplicate assertion.